### ISSUE TITLE
Update Pull Request Template to reflect implementers

### ISSUE THREAD

**Issue by marcoscaceres** [w3c/html-aam#220]
Right now, the pull request template lists a bunch of web engines as potential implementers of this spec... however, assistive technologies are also able to implement this spec, so it would be prudent to add them to the template. 

So, could the community here help us identify implementers, help us file bugs on implementations, and track implementation commitments?

**Comment by marcoscaceres** [comment]
This is the template: https://github.com/w3c/html-aam/blob/gh-pages/.github/PULL_REQUEST_TEMPLATE.md

**Comment by stevefaulkner** [comment]
Hey @marcoscaceres, while it is possible for AT to implement the spec independently of browsers it is not the intention and I do not know of any that do or are planning to. AT are consumers of the information exposed by browser via the acc API's

 > assistive technologies are also able to implement this spec

**Comment by marcoscaceres** [comment]
I see (I don't actually... as I've no idea how this specs works or who is supposed to use this spec 🤪) - so what should the template actually look like?  

From https://github.com/w3c/html-aam/pull/219 it seemed that "Orca" was implementing... who else would implement? Would/does Firefox?

**Comment by LJWatson** [comment]
The AAM describes the way elements and attributes are mapped to the different platform acc API, then verifies each mapping in at least two browser implementations. So the audience is primarily the owners responsible for the acc API and their implementations in browsers.

**Comment by stevefaulkner** [comment]
@marcoscaceres
> From #219 it seemed that "Orca" was implementing... 

@joanmarie is ORCA implementer as well as implementing acc API stuff in webkit etc. I suggest that what is meant is that once support lands in browsers, support for what is exposed in browsers will then need to be added to ORCA and other SR's

**Comment by marcoscaceres** [comment]
ok, so WebKit can stay in the template. I'd be interested to see how one of these is implemented in WebKit and see how it's done in Firefox too.

**Comment by joanmarie** [comment]
@marcoscaceres as a general rule, it's a mere few lines of code here and there. Role/element mapping is tends to be simple.

**Comment by joanmarie** [comment]
@marcoscaceres 

> also able to implement this spec

Wellllll....

As has already been stated, the AAMs specify what *users agents* MUST do in their code so that the information the ATs receive makes sense. In many instances, what an AT needs to do to add support is little to nothing.

Were it me, I'd probably close this as WONTFIX on the grounds that user agents are the implementors.

I also fear that I caused confusion by not limiting myself to screen reader development but also contributing to the three major user agents. If that's the problem, my apologies. :wink: 

P.S. @stevefaulkner. Orca; not ORCA. :slightly_smiling_face:

**Comment by marcoscaceres** [comment]
> If that's the problem, my apologies.

Totally not a problem! That’s amazing 🤩. Please do ping me when you send a patch to Gecko (:mcaceres on bugzilla).
